src/style.css

```diff
@@ -1 +1,82 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 96px;
+  height: 116px;
+  margin: 22px;
+}
+
+.stars__line {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('images/star.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.stars__star::before {
+  content: '';
+  display: block;
+  width: 0;
+  height: 100%;
+  background-image: url('images/star-active.svg');
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 1;
+}
+
+/* .stars--0::before {
+  width: 0%;
+} */
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('images/star-active.svg');
+}
+
+/* .stars--1::before {
+  width: 20%;
+} */
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('images/star-active.svg');
+}
+
+/* .stars--2::before {
+  width: 40%;
+} */
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('images/star-active.svg');
+}
+
+/* .stars--3::before {
+  width: 60%;
+} */
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url('images/star-active.svg');
+}
+
+/* .stars--4::before {
+  width: 80%;
+} */
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('images/star-active.svg');
+}
+
+/* .stars--5::before {
+  width: 100%;
+} */
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('images/star-active.svg');
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead of styling the `.stars__star` element directly within the `.stars--{number}` class, consider adding a modifier class to the star elements that need different styling. This will make your CSS more maintainable and scalable.